docs/docs/guides/integrate/identity-providers/openldap.mdx

```diff
@@ -148,6 +148,36 @@ The user can be applied after OpenLDAP is running with
 ldapadd -x -h localhost -D "cn=admin,dc=example,dc=com" -f example.ldif -w 'Password1!'
 ```
 
+### Docker Compose configuration
```

Comment:
Looks good to me, could you please combine it with the installation guide of OpenLDAP to give different options. https://github.com/zitadel/zitadel/blob/b63c5fdb1792603263cd8c625d06a0498dbefe40/docs/docs/guides/integrate/identity-providers/openldap.mdx?plain=1#L37